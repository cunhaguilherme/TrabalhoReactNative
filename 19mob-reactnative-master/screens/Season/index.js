import React, { Component } from 'react';

import { Text, View } from 'react-native';

class Season extends Component {
    render() {
        console.log(this.props);
        return (
            <View>
                <Text>Oi eu sou a tela de Season</Text>
            </View>
        );
    }
}

export default Season;