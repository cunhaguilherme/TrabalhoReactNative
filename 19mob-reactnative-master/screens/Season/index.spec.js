import React from 'react';
import { render } from 'react-native-testing-library';

import Season from './index.js';

test('should be render screen', async () => {
    const {baseElement} = render(<Season />);
    console.log(baseElement);
    expect(baseElement).toMatchSnapshot();
});