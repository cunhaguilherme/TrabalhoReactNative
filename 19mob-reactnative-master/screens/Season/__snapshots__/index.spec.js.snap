// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should be render screen 1`] = `undefined`;
